### ISSUE TITLE
problems in re-implement your experiment.

### ISSUE THREAD

**Issue by wangxi123** [MhLiao/TextBoxes#36]
Hi,MhLiao.
I have  implemented your TextBoxes on SynthText and icdar2013.but here are some problems that bother me.i got the result **recall =0.488,precision = 0.938 and f-measure=0.64** for 700x700 single scale in icdar test set. For multi scale the f-measue is 0.73. that seems worse than your result.
the detail information for my training is below:
_step 1: pretrain on the synthText data._

- pretrain model:the VGG_ILSVRC_16_layers_fc_reduced.caffemodel

- train data: about 85w SynthText , train size:700x700，batch size:8 (GPU limit)

- lr: 0.0001 for 6w iterations, 0.00001 for the rest 12w iterations. total 18w iterations(**loss about 2.0**).  
_step 2: train on the icdar2013 train data_

- pretrain model:the model of step 1.

- train size:700x700,batch_size = 4.

- lr :0.00001 for 3k iterations.(**loss about 1.5**)
I have tried other settings: train data resize to 500x500 but still got a low recall(about 0.46) .by the way,the final loss is down to about 2.0 when trained on the synthdata, i don't know whether i have not 
taken into account.
looking forward to your reply.

**Comment by MhLiao** [comment]
@wangxi123 Generally，training the SynthText and ICDAR 2013 with 300*300 is enough, and the performance is even better if 700*700 training is applied after 300*300. For SynthText, I trained it 50k iterations with batch size 32.  For ICDAR 2013 dataset, I trained it 2k iterations with batch size 32.
Note that the parameter "min_dim" should not be changed when you change your training input size.

**Comment by wangxi123** [comment]
@MhLiao Thanks,i will have a try.by the way,what your final loss about when you train SynthText for 300*300 size? I always feels that the loss of my training is too high...

**Comment by ZDDEAN** [comment]
> 嗨，MhLiao。
> 我已经在SynthText和icdar2013上实现了你的TextBoxes。但是这里有一些困扰我的问题。我在icdar测试集中得到了700x700单一刻度的结果**召回率= 0.488，精度= 0.938和f-measure = 0.64**。对于多尺度，f-测量值为0.73。这似乎比你的结果更糟糕。
> 我的培训的详细信息如下：_第1步：预先_训练
> _synthText数据。_
> 
> * pretrain模型：VGG_ILSVRC_16_layers_fc_reduced.caffemodel
> * 列车数据：约85w SynthText，列车大小：700x700，批量大小：8（GPU限制）
> * lr：0.0001用于6w迭代，0.00001用于其余12w迭代。总共18w迭代（**损失约2.0**）。
>   _第2步：训练icdar2013列车数据_
> * pretrain模型：步骤1的模型。
> * 列车大小：700x700，batch_size = 4。
> * lr：0.00001用于3k次迭代。（**损失大约1.5**）
>   我尝试了其他设置：列车数据调整大小为500x500但仍然有较低的召回率（约0.46）。顺便说一句，最终损失降至约2.0，当受过训练时synthdata，我不知道我是否没有
>   考虑到。
>   期待你的回复。

您能分享一下synthtext格式转换为icder格式的脚本吗？谢谢鸭！

**Comment by ZDDEAN** [comment]
> 嗨，MhLiao。
> 我已经在SynthText和icdar2013上实现了你的TextBoxes。但是这里有一些困扰我的问题。我在icdar测试集中得到了700x700单一刻度的结果**召回率= 0.488，精度= 0.938和f-measure = 0.64**。对于多尺度，f-测量值为0.73。这似乎比你的结果更糟糕。
> 我的培训的详细信息如下：_第1步：预先_训练
> _synthText数据。_
> 
> * pretrain模型：VGG_ILSVRC_16_layers_fc_reduced.caffemodel
> * 列车数据：约85w SynthText，列车大小：700x700，批量大小：8（GPU限制）
> * lr：0.0001用于6w迭代，0.00001用于其余12w迭代。总共18w迭代（**损失约2.0**）。
>   _第2步：训练icdar2013列车数据_
> * pretrain模型：步骤1的模型。
> * 列车大小：700x700，batch_size = 4。
> * lr：0.00001用于3k次迭代。（**损失大约1.5**）
>   我尝试了其他设置：列车数据调整大小为500x500但仍然有较低的召回率（约0.46）。顺便说一句，最终损失降至约2.0，当受过训练时synthdata，我不知道我是否没有
>   考虑到。
>   期待你的回复。

您能分享一下synthtext格式转换为icdar格式的脚本吗？谢谢鸭！
